wolfcrypt/src/sp_cortexm.c

```diff
@@ -60,6 +60,9 @@
 #define __asm__        __asm
 #define __volatile__   volatile
 #endif
+#ifdef WOLFSSL_ZEPHYR
```

Comment:
Can you tell me the error this resolves? Is it the new `register sp_digit* r asm ("r0") = (sp_digit*)r_p;` type expressions? I think we may want to fix this differently (not just for Zephyr).
Note: This file is generated, so we must also update an internal script.

Reply:
Yes, these declarations all fail using the Zehpyr SDK. I'm not sure currently whether other compilers are also affected (e.g., the `arm-none-eabi` toolchain). 
Maybe it's better to generally change the declarations to use `__asm__` instead of `asm`? I didn't realize that the file is generated. Hence, I'm not sure if I can do such a fix?

Reply:
Yeah we'll need to fix this issue on our side. Feel free to keep it for now. We've got a PR open to address similar issues with IAR, so we will integrate a proper update there... #6943.

Reply:
Hi @Frauschi , this has been fixed in #6943. Please revert this change and confirm 6943 solves your compilation issues. Thanks